### ISSUE TITLE
Add support for table and column comments.

### ISSUE THREAD

**Issue by cansjt** [laughingman7743/PyAthena#263]
Though Athena supports them, as of today, there are no ways to set comments in a table definition. Neither on the the table itself or on the table's columns.

We'd like to be capable of doing that, with SQLAlchemy, that defines the interfaces needed to implement this.

Example:
```python
from sqlalchemy.sql.ddl import CreateTable
from sqlachemy import Column, Integer, MetaData, Table

from pyathena.sqlachemy_athena import AthenaDialect


table = Table('table_name',
              MetaData(),
              Column('c', Integer, comment='Some very important data'),
              comment='A very business critical table',
              schema='s')
dialect = AthenaDialect()
print(str(dialect.ddl_compiler(dialect, CreateTable(table))))
```
Would produce the following SQL statement:
```sql
CREATE EXTERNAL TABLE s.table_name (
    c INTEGER COMMENT 'Some very important data'
)
COMMENT 'A very business critical table'
```

References:
- [Athena's `CREATE TABLE` statement documentation](https://docs.aws.amazon.com/athena/latest/ug/create-table.html)

**Comment by laughingman7743** [comment]
Thanks for the pull request.

I would not accept any way to parse the result of DESCRIBE TABLE. If there is any change in the output, an error will occur. If you want to get the comments of the table, you can use the information schema.
https://docs.aws.amazon.com/athena/latest/ug/querying-glue-catalog.html

I'm not sure if the location and compression settings can be retrieved from the information schema, but Boto3 can retrieve the table metadata.
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/athena.html#Athena.Client.get_table_metadata
If we can't get this information from the information schema, I think it's best to use the API, because inside the cursor we are just sending a request to the AWS API, and it's easy to implement to get the data from this API.

**Comment by laughingman7743** [comment]
Here is the branch where I tried to implement the table metadata retrieval in the fix of #244.
https://github.com/laughingman7743/PyAthena/pull/265
Please check it out.
